src/libAtomVM/term.h

```diff
@@ -1334,30 +1346,19 @@ static inline int term_list_length(term t, int *proper)
  * @param b second term
  * @return 1 if they are the same, 0 otherwise.
  */
-static inline int term_exactly_equals(term a, term b, Context *ctx)
+static inline int term_exactly_equals(term a, term b, GlobalContext *global)
```

Comment:
Should we make this return a bool, while we are changing the signature of the function?